pkg/manifests/manifests_test.go

```diff
@@ -258,9 +259,7 @@ func TestManifests(t *testing.T) {
 			metricsCertSecretName, svc.Annotations[ServingCertSecretAnnotation])
 	}
 
-	if _, err := f.RouterServiceCloud(ci); err != nil {
-		t.Errorf("invalid RouterServiceCloud: %v", err)
-	}
+	manifests.LoadBalancerService()
```

Comment:
Delete the `manifests.` (this is in the manifests package).

Reply:
Fixed